src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -138,8 +133,8 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Build, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL" />
-    <Reference Include="Microsoft.Build.Framework, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.Build" />
```

Comment:
Locking to a version number instead of take-latest created version conflicts due to new dependencies flowing throughout the solution from nuget packages in VisualStudio14.Package. Not breaks but compiler warnings. Take-latest means a version jump of 2 (4->14 is only 2 versions, believe it or not).